spec/parts/linux/cloud-init/artifacts/imds_mocks/network/multi_nic.json

```diff
@@ -0,0 +1 @@
+[{"ipv4":{"ipAddress":[{"privateIpAddress":"0.0.0.0","publicIpAddress":""}],"subnet":[{"address":"10.10.10.10","prefix":"27"}]},"ipv6":{"ipAddress":[]},"macAddress":"AAAA525BA3A7"},{"ipv4":{"ipAddress":[{"privateIpAddress":"1.1.1.1","publicIpAddress":""}],"subnet":[{"address":"11.11.11.11","prefix":"27"}]},"ipv6":{"ipAddress":[]},"macAddress":"7C1E525AAAAA"}]
```

Comment:
It's a json file? Although the file extension doesn't really matter in this test case

Reply:
yes it's a json file, since IMDS outputs json